pkg/neorpc/result/peers.go

```diff
@@ -68,6 +72,28 @@ func (p *Peers) addPeers(addrs []string) {
 	}
 }
 
+// addConnectedPeers adds a set of connected peers to the given peer slice.
+func (p *Peers) addConnectedPeers(connectedPeers []network.PeerInfo) {
+	for i := range connectedPeers {
+		host, portStr, err := net.SplitHostPort(connectedPeers[i].Address)
+		if err != nil {
+			continue
+		}
+		port, err := strconv.ParseUint(portStr, 10, 16)
+		if err != nil {
+			port = 0
+		}
```

Comment:
It would be nice to unify this snippet with the same one from `(p *Peers) addPeers`.

Reply:
@AnnaShaleva I'll add it as a helper in `peers.go` since I didn't find the exact snippet anywhere else, but let me know if it should be moved.

Reply:
It's the proper place since the only usage of this helper is `peers.go` itself.